src/Infrastructure/LeanCode.Kratos/KratosAuthenticationHandler.cs

```diff
@@ -26,6 +26,18 @@ public class KratosAuthenticationHandler<TOptions> : AuthenticationHandler<TOpti
 
     private readonly IFrontendApi api;
 
+#if NET8_0_OR_GREATER
```

Comment:
Caaan we get rid of this ifdefs? We don't need them, really, since we will target .NET 8 onwards only here.